components/top-nav/src/index.js

```diff
@@ -0,0 +1,238 @@
+  fontWeight: 800,
+  alignItems: 'center',
+  justifyContent: 'flex-start',
+  fontSize: '30px',
+  ' a': {
+    display: 'flex',
+    color: WHITE,
+    textDecoration: 'none',
+    marginBottom: 0,
+    paddingBottom: 0,
+    borderBottom: '1px solid transparent',
+    ':hover': {
+      borderBottom: `1px solid ${WHITE}`,
+    },
+  },
+  ' h1': {
+    fontSize: '30px',
+  },
+  ' svg': {
+    marginRight: SPACING.SCALE_2,
+  },
+});
+
+const LogoSearchWrapper = glamorous.div({
+  display: 'flex',
+  justifyContent: 'space-between',
+  [MEDIA_QUERIES.LARGESCREEN]: {
+    display: 'block',
+    paddingRight: SPACING.SCALE_3,
+    width: '33.33%',
+  },
+});
+
+const RightHandSide = glamorous.div({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'flex-start',
+  marginTop: SPACING.SCALE_2,
+  ' a': {
+    border: 0,
+    textDecoration: 'none',
+    borderBottom: '1px solid transparent',
+    marginBottom: 0,
+    paddingBottom: 0,
+    ':hover': {
+      borderBottom: `1px solid ${WHITE}`,
+    },
+  },
+  [MEDIA_QUERIES.LARGESCREEN]: {
+    ' button': {
+      display: 'none',
+    },
+    width: '66.66%',
+    paddingLeft: SPACING.SCALE_3,
+    marginTop: 0,
+  },
+});
+
+const SearchWrapper = glamorous.div({
+  marginTop: SPACING.SCALE_1,
+  width: '50%',
+  [MEDIA_QUERIES.LARGESCREEN]: {
+    width: '100%',
+  },
+});
+
+const Ul = glamorous.ul(({
+  serviceTitle,
+}) => ({
+  display: 'flex',
+  flexWrap: 'wrap',
+  margin: 0,
+  padding: 0,
+  marginTop: SPACING.SCALE_1,
+  width: '100%',
+  [MEDIA_QUERIES.LARGESCREEN]: {
+    paddingTop: serviceTitle ? 0 : SPACING.SCALE_5,
+    width: 'auto',
+  },
+}));
+
+const Li = glamorous.li(({
+  active,
+}) => ({
+  flex: '1 0 50%',
+  width: '100%',
+  listStyleType: 'none',
+  margin: 0,
+  padding: '3px 0',
+  borderBottom: `1px solid ${PROPOSITION_BORDER}`,
+  [MEDIA_QUERIES.LARGESCREEN]: {
+    borderBottom: 0,
+    flex: 'none',
+    width: 'auto',
+    paddingRight: SPACING.SCALE_3,
+  },
+  ' a': {
+    color: active ? LIGHT_BLUE : WHITE,
+    textDecoration: 'none',
+    borderBottom: '1px solid transparent',
+    fontWeight: 700,
+    ':hover': {
+      borderBottomColor: active ? LIGHT_BLUE : WHITE,
+    },
+  },
+}));
+
+const TopNav = ({
+  active,
+  bgColor,
+  color,
+  company,
+  serviceTitle,
+  search,
+  children,
+  ...props
+}) => (
+  <React.Fragment>
+    <TopNavWrapper bgColor={bgColor} color={color} {...props}>
+      <TopNavInner>
+        <LogoSearchWrapper>
+          <Company>
+            {company}
+          </Company>
+          {search &&
+            <SearchWrapper>
+              {search}
+            </SearchWrapper>
+          }
+        </LogoSearchWrapper>
+        <RightHandSide>
+          {serviceTitle}
+          {children &&
+          <React.Fragment>
+            <Button mb={0}>menu</Button>
+            <Ul serviceTitle={serviceTitle}>
+              {children.length && children.map ? (
+                children.map((child, i) => (
```

Comment:
this is where the context api and render props would come in useful, perhaps create a github issue to research this and add a `TODO: #XXX` to the code